Run7/step14/dark_shutterOpen.cfg

```diff
@@ -0,0 +1,7 @@
+#  20 15sec DARK images taken with room lights off and Shutter OPEN
+[ACQUIRE]
+dark
+[DARK]
+dark=15.0 20     # integration time and image count for dark set
+SHUTTER=OPEN
```

Comment:
Should `SHUTTER = OPEN` come before `dark=15.0 20`?

Reply:
Based on the parsing of `fp-scripts/acquire.py` arguments, it should not matter the order of the keywords, so long as they are within the same block. Unless I am misunderstanding something?

Reply:
I'm just asking a question, if that's ok.  If the order doesn't matter, that's fine.  I didn't write this code, but there are cases where the order does matter, e.g., in the flat pair acquisitions, so I wanted to ask to be sure.  Please ask someone else to review your changes in the future.

Reply:
Thank you for asking a question, and I apologize for not clarifying. I am not trying to be defensive or combative, and I understand that it reads that way. I apologize for my poor choice of words.

I was asking if I was misunderstanding because we have been fairly arbitrary with the ordering of keywords, and I was curious if the order did impact things.

Again, I apologize for being curt. That is not my intention.